be-typescript-express/files/src/index.ts

```diff
@@ -0,0 +1,7 @@
+var express = require("express")
+
+const app = express();
+
+app.listen(3000, () => {
```

Comment:
Hey @sino92 , it would be also better to add a simple response to the root url. It is the easiest way to test the quickstarter after it is deployed. I guess otherwise it will responds with a  'Cannot get /' message by default.

something like this will be enough i think,
`app.get('/', (req, res) => res.send('Hello World!'))`